core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -80,6 +80,7 @@ class ContainerPool(
     private val authStore = WhiskAuthStore.datastore(config)
     setVerbosity(verbosity)
 
+    val mounted = !standalone
```

Comment:
There are several places in `ContainerPool.scala` where `!standalone` are used. Want to replace those with `mounted` as well? (They distinguish `mounted` as well)